documentation/CAMARA-Security-Interoperability.md

```diff
@@ -0,0 +1,277 @@
+
+The client MUST authenticate with the authorization server as described in [Client Authentication Section](#client-authentication).
+
+## Format of `login_hint`
+
+This CAMARA document clarifies the values used in login_hint in the following way:
+
+    * **_tel_**
+
+      For phone numbers. The `login_hint` must be a tel URI as defined in [RFC 3966](https://www.rfc-editor.org/info/rfc3966) for global phone numbers without visual separators in [E.164](https://www.itu.int/rec/T-REC-E.164-201011-I/en) format. For example, `tel:+34666666666`.
+
+    * **_ipport_**
+
+      For IPv4 and IPv6 addresses, that can optionally include a port. For example, `ipport:80.90.34.2:16790`, `ipport:80.90.34.2`, `ipport:[2001:db8::1]:8080` or `ipport:[2001:db8::1]`.
+
+
+## Offline Access
+
+#### Refresh Token Issuance
+
+Neither OIDC, CIBA, nor OAuth2 define a way for clients to indicate whether they need a refresh_token. 
+Refresh token issuance is optional and at the discretion of the AZ.
+
+Camara uses the scope `offline_access` in the authorization request to indicate to the AZ that the client requests a refresh token additionally to the access token for Camara API access.
+
+---
+**NOTE**
+
+OIDC defines the scope `offline_access` and it is used to get an access token to be used at the UserInfo endpoint, and also to obtain a request token to refresh this access token.
+
+---
+
+The OIDC rules for `offline_access` as defined in [OIDC Offline Access](https://openid.net/specs/openid-connect-core-1_0.html#OfflineAccess) apply. 
+
+The OIDC section on [OIDC Using Refresh Tokens](https://openid.net/specs/openid-connect-core-1_0.html#RefreshTokens) is not changed by this document.
+
+The OIDC security and privacy considerations regarding offline access and refresh tokens apply e.g. on
+[Token lifetime](https://openid.net/specs/openid-connect-core-1_0.html#TokenLifetime) and [Offline Access Privacy](https://openid.net/specs/openid-connect-core-1_0.html#OfflineAccessPrivacy).
+
+### Refresh Token Usage
+
+This section applies to OIDC Authorization Code Flow and CIBA. Other flows do not need refresh tokens and MUST not use refresh tokens.
+
+In addition to [OIDC Using Refresh Tokens](https://openid.net/specs/openid-connect-core-1_0.html#RefreshTokens) and [OAuth2 Refreshing an Access Token](https://datatracker.ietf.org/doc/html/rfc6749#section-6) this document defines the following:
+
+    The authorization decisions for issuing an access token MUST be re-checked by the Authorization Server when a refresh token is used. 
+        * A new access token MUST not be issued if the user has revoked their consent.
+        * A new access token MUST not be issued if the client status regarding the requested API access has changed.
```

Comment:
```suggestion
        * A refresh token MUST be revoked if the client status regarding the requested API access has changed.
```

Reply:
The sentence before the list is `The authorization decisions for issuing an access token MUST be re-checked by the Authorization Server when a refresh token is used. `. So, I would argue that the list is about `access tokens`.

I think it is an implementation detail whether the refresh token is revoked in the moment the user revokes their consent or whether the AZ return invalid_grant instead of issuing a new access token.